### ISSUE TITLE
Can u upgrade flutter_cache_manager  version??

### ISSUE THREAD

**Issue by jinfagang** [Baseflow/flutter_cached_network_image#544]
```
Because cached_network_image >=2.5.0 depends on flutter_cache_manager ^2.1.0 which depends on image ^2.1.18, cached_network_image >=2.5.0 requires image ^2.1.18.
And because image >=2.1.14 <3.0.0-nullsafety.0 depends on xml ^4.2.0 and flutter_svg >=0.20.0-nullsafety.0 depends on xml ^5.0.0-nullsafety.0, cached_network_image >=2.5.0 is incompatible with flutter_svg >=0.20.0-nullsafety.0.
So, because flutter_app depends on both flutter_svg ^0.20.0-nullsafety.3 and cached_network_image ^2.5.0, version solving failed.
```

Obviously, flutter_cache_manager you constraint is too low now. It dependes xml version also too low. At least using xml 5+ ....

**Comment by jinfagang** [comment]
BTW, flutter_cache_manager  official version is now 2.1.1 which of course developed by you guys. So please help us upgrade the base package version to compatible with flutter_svg or xml

**Comment by renefloor** [comment]
It isn't fixed on 2.1.1, but on `^2.1.1`, which means anything of at least 2.1.1, but lower than 3.0.0. At the moment there is no flutter_cache_manager of version 3.0.0, so that's not the problem here.

The problem is that flutter_cache_manager has dependencies on `image: ^2.1.18`.

Upgrading that image dependency to `3.0.0-nullsafety.0` is part of the nullsafety migration on which we are working. However, it wouldn't make sense using a pre-release dependency in a stable release, so we can't just increase the maximum dependency and leave it that way.

You can either downgrade your flutter_svg dependency or override image and hope for the best: https://dart.dev/tools/pub/dependencies#dependency-overrides
